models/prod/final/org/farmer_silt_vulnerable_org.sql

```diff
@@ -0,0 +1,16 @@
+{{ config(
+  materialized='table'
+) }}
+
+SELECT DISTINCT *
+FROM {{ ref('farmer_silt_vulnerable_niti_union') }}
+
+UNION
+
+SELECT DISTINCT *
+FROM {{ ref('farmer_silt_vulnerable_gramin') }}
+
```

Comment:
what is the motivation for creating these models ? 

I see that both the ref models used `farmer_silt_vulnerable_niti_union ` and `farmer_silt_vulnerable_gramin ` are groping by to compute a metric (total_silt_carted) across various dimensions of locaiton/ngo. 

So if the dimensions are same across location/ngo, but the metric is different. So in this case , are we saying that keep both metric counts (which defeats the purpose of distinct i guess ? Shouldn't we be doing some kind of avg here or aggregate. 

Same question across other models where we are doing similar stuff.

Reply:
we aren't doing averages or aggregates here because - 
they want to be able to filter by project, state, district, taluka, village and dam. and right now with the way the model is written, we will have one row for each unique value of any combination thereof so for the dashboard requirements this is how we need it. If you look at how siddhant aggregated the models for gdgs_union or niti_union, you will see he used the same logic for the same reason :)